core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -194,20 +194,30 @@ public URIStatus getStatus(AlluxioURI path, GetStatusOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     // TODO(calvin): Fix the exception handling in the master
     try {
-      return masterClient.listStatus(path);
+      return masterClient.listStatus(path, options);
     } catch (FileDoesNotExistException e) {
       throw new FileDoesNotExistException(ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage(path));
     } finally {
       mContext.releaseMasterClient(masterClient);
     }
   }
 
+  /**
+   * @inheritDoc
```

Comment:
done. I copied the style from other file (SwfitUnderFileSystem). I have changed that too